docs/checks/promql/offset.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+parent: Checks
+grand_parent: Documentation
+---
+
+# promql/offset
+
+This check will warn if a metric selector name matches a configured prefix and doesn't have a minimum offset.
+Some cloud monitoring services (e.g. AWS CloudWatch) metrics' have a delay before reaching Prometheus,
+so queries on these metrics should be used with an offset that matches the delay to look at the "past".
+
+## Configuration
+
+Syntax:
+
+```js
+offset {
+  prefix = "$pattern"  
+  min    = "###u"
+}
+```
+
+- `prefix` - regexp pattern to match metric name prefix on,
+- `min`    - minimum duration the look back offset must be set to, `10m` would mean 10 minutes
+
+## How to enable it
+
+This check is not enabled by default as it requires explicit configuration
+to work.
+To enable it add one or more `rule {...}` blocks and specify all required
+rules there.
+
+Examples:
+
+Ensure that all metrics beginning with `aws_` have a minimum offset of 5 minutes:
+
+```js
+rule {
```

Comment:
Maybe add an example for multiple offsets

Reply:
Good catch. I'll add those in.